README.md

```diff
@@ -87,8 +88,8 @@ res1: AppSettings = AppSettings(HttpSettings(ServerSettings(localhost,8080),Some
 scala> parser.decodePathF[IO, ServerSettings]("http.server")
 res2: cats.effect.IO[ServerSettings] = IO(ServerSettings(localhost,8080))
 
-scala> parser.decodePathF[IO, ServerSettings]("path.not.found")
-res3: cats.effect.IO[ServerSettings] = IO(throw io.circe.ParsingFailure: Path not found in config)
+scala> parser.decodePathF[IO, ServerSettings]("path.not.found").option.unsafeRunSync
+res3: Option[ServerSettings] = None
```

Comment:
Instead of `IO(throw io.circe.ParsingFailure: Path not found in config)` we now have `IO(...)`. I thought `None` would be better, happy to change if needed

Reply:
@vhiairrassary I think we can give it a little more information when reading it by doing `.attempt.unsafeRunSync()` which will give us the Either with the exception in it.  

Sorry that this didn't get looked at sooner!

Reply:
Done in 8c5fbd5.